Resources/public/mapbender.element.digitizer.js

```diff
@@ -998,7 +998,9 @@
                 schema.elementsTranslated = true;
             }
 
-            DataUtil.eachItem(widget.currentSettings.formItems, function(item) {
+            var formItems = JSON.parse(JSON.stringify(widget.currentSettings.formItems)); // Deep clone
```

Comment:
Please use jQuery extend ( https://api.jquery.com/jquery.extend/ ) . JSON.parse(JSON.stringify()) can cause information lose,  and is very slow.